Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -5298,15 +5298,35 @@ ElementInfo IWindowExplorer.LearnElementInfoDetails(ElementInfo EI, PomSetting p
                 EI.ElementTypeEnum = elementTypeEnum.Item2;
             }
 
-            if ((string.IsNullOrEmpty(EI.XPath) || EI.XPath == "/") && EI.ElementObject != null)
+
+            if(pomSetting == null)
```

Comment:
pomsettings  is not being used in if or else block then why we need condition for it?

Reply:
pomsettings is null when the function is called through live spy